### ISSUE TITLE
Revisions?

### ISSUE THREAD

**Issue by jhodges10** [dash-docs/dash-docs#43]
Source File: _includes/devdoc/dash-core/rpcs/rpcs/gobject.md

Not sure what `revision` is in `gobject prepare`. It's not something I've seen before and I'm wondering if it should be `type` instead.

**Comment by thephez** [comment]
It is the second parameter as named in the Dash Core source. See:
 - https://github.com/dashpay/dash/blob/master/src/rpc/governance.cpp#L130
 - https://github.com/dashpay/dash/blob/master/src/rpc/governance.cpp#L144

It appears that it is typically set to 1 (based on the examples I have seen).

**Comment by UdjinM6** [comment]
Yep, governance system was designed to have multiple revisions sometimes later but this param is not used in any meaningful way (yet?) and is expected to always be `1` for now.

**Comment by jhodges10** [comment]
Thanks for clarifying @UdjinM6 and @thephez. 

Pretty cool to see this as I didn't know `gobject` revisions were ever something being considered. Was the thinking that you could update the governance object and then change the amount requested or the payment address (potentially to adjust for escrow or adjusted multi-month)?

I imagine this is something that would require a great deal of transparency on for the voters so as not to be confused by a vote cast on one version of the `gobject` vs another.
